march_shared_classes/src/march_shared_classes/gait/joint_trajectory.py

```diff
@@ -10,12 +10,13 @@ class JointTrajectory(object):
     """Base class for joint trajectory of a gait."""
 
     setpoint_class = Setpoint
+    digits = 4
 
     def __init__(self, name, limits, setpoints, duration, *args):
         self.name = name
         self.limits = limits
         self._setpoints = setpoints
-        self._duration = duration
+        self._duration = round(duration, self.digits)
```

Comment:
```suggestion
        self._duration = round(duration, Setpoint.digits)
```
This way we only have to manage the variable in the setpoint class